examples/js/controls/VRControls.js

```diff
@@ -47,17 +53,38 @@ THREE.VRControls = function ( object, onError ) {
 
 			var vrInput = vrInputs[ i ];
```

Comment:
Should there be iteration through `vrInputs` anymore?  I understand that this is how it's been, but it never made sense to me, wouldn't the update always be based on the last `PositionSensorVRDevice` pushed to the array?  It makes even less sense now that `VRDisplay` handles pose too - in `VREffect` we use the first `VRDisplay`, but `VRControl` will update based on the last `VRDisplay`... I'd suggest dropping the `vrInputs` array and pick the first device in both the new and deprecated code paths.